src/util/get-change.ts

```diff
@@ -0,0 +1,16 @@
+// We want to get the % changed from the baseline value
+// [Same]  | pinned:10 | result: 10 | 0
+// [Faster]| pinned:10 | result: 8 | -20
+// [Slower]| pinned:10 | result: 12 | +20
```

Comment:
Might be best to update the references to `pinned` and `result` here – especially as the `pinned`->`baseline` and `result`->`target` translation could be confusing